<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicTimerWidget</class>
 <widget class="QWidget" name="MusicTimerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>定时设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>470</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>定时设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>45</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="first">
    <widget class="QRadioButton" name="noSetRadioButton1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>不设置定时开始播放</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="setRadioButton1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置定时开始播放</string>
     </property>
    </widget>
    <widget class="QComboBox" name="hourComboBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="secComboBox1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="repeatComboBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_F">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>播放时间:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>135</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>重复频率:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>播放列表:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>222</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>播放曲目:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="plistComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>188</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="psongComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="second">
    <widget class="QRadioButton" name="noSetRadioButton2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>不设置定时停止播放</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="setRadioButton2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置定时停止播放</string>
     </property>
    </widget>
    <widget class="QComboBox" name="hourComboBox2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="secComboBox2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="repeatComboBox2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_S">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>停止时间:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_S2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>135</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>重复频率:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="third">
    <widget class="QRadioButton" name="noSetRadioButton3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>不设置定时关机</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="setRadioButton3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置定时关机</string>
     </property>
    </widget>
    <widget class="QComboBox" name="hourComboBox3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="secComboBox3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>92</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="repeatComboBox3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_T">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>关机时间:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_T2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>135</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>重复频率:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="backgroundMask1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>125</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,200)</string>
   </property>
   <widget class="QPushButton" name="timerToPlay">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>定时播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerToStop">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>75</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>定时停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timerToShutdown">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>120</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>定时关机</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>317</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>317</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>320</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask2" native="true">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>45</y>
     <width>345</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask2</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>backgroundMask1</zorder>
  <zorder>confirm</zorder>
  <zorder>cancel</zorder>
  <zorder>line_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
